src/adapter/graphql/src/queries/accounts/account_flow_runs.rs

```diff
@@ -10,22 +10,22 @@
 use std::collections::HashSet;
 
 use futures::TryStreamExt;
-use kamu_accounts::{Account, AuthenticationService};
+use kamu_accounts::Account as AccountEntity;
 use kamu_flow_system as fs;
 
 use crate::prelude::*;
-use crate::queries::{Flow, FlowConnection, InitiatorFilterInput};
+use crate::queries::{Dataset, DatasetConnection, Flow, FlowConnection, InitiatorFilterInput};
 
 pub struct AccountFlowRuns {
-    account: Account,
+    account: AccountEntity,
 }
 
 #[Object]
 impl AccountFlowRuns {
```

Comment:
I\m looking at both queries right now, and I do not understand where do we have security checks. I don't think we should allow querying this information for any other account then the currently logged in.

Reply:
This is a query, not a mutation. There is no security check in `Account` query.

Reply:
Found